docs/zh/latest/plugins/openid-connect.md

```diff
@@ -58,7 +58,7 @@ OAuth 2 / Open ID Connect（OIDC）插件为 APISIX 提供身份验证和自省
 ![token introspection](../../../assets/images/plugin/oauth-1.png)
 
 以下是 curl 命令，用于将插件启用到外部服务。
-通过自省请求标头中提供的令牌，此路由将保护 https://httpbin.org/get（echo 服务）。
+通过自省请求标头中提供的令牌，此路由将保护 `https://httpbin.org/get`（echo 服务）。
```

Comment:
Hi, we may have many cases like this. 😄 Would you like to have a discussion with @haifeng9414, @kwanhur, and @navendu-pottekkat? They're working on Documentation Style unify.

Reply:
No

Reply:
> 嗨，我们可能有很多这样的情况。😄您想与@haifeng9414、 @ kwanhur和@navendu-pottekkat 进行讨论吗？他们正在研究文档样式统一。
![image](https://user-images.githubusercontent.com/16237905/159879213-1278a2dc-8e60-49cf-81bf-7222f8e6bb42.png)
This should be caused by the markdown hyperlink format